packages/waku/src/vite-rsc/cli.ts

```diff
@@ -28,6 +28,7 @@ export async function cli(cmd: string, flags: Record<string, any>) {
       server: {
         port: parseInt(flags.port || '3000', 10),
       },
+      base: config?.basePath ?? '/',
```

Comment:
Technically Vite's `base` and Waku's "base" can mean different things at least during build. Vite's `base` is where the client static assets are available https://vite.dev/config/shared-options.html#base (e.g. different cdn domain), but Waku's "base" is the "base pathname" which Waku routing assumes. The latter concept is called `basename` on React router https://reactrouter.com/api/declarative-routers/Router#basename.

Though I don't think such differentiation is necessary for initial goal, I thought it's worth mentioning.